src/runtime/operator.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Python.Runtime
+{
+    internal static class OperatorMethod
+    {
+        public static Dictionary<string, Tuple<string, int>> OpMethodMap { get; private set; }
+
+        private static Dictionary<string, string> _pyOpNames;
+        private static PyObject _opType;
+
+        static OperatorMethod()
+        {
+            // TODO: Rich compare, inplace operator support
+            OpMethodMap = new Dictionary<string, Tuple<string, int>>
```

Comment:
This needs explicit key comparer for string keys, the Ordinal one. Otherwise it might fail to work under weird locales.

Reply:
Yes, from [How to compare strings in C#](https://docs.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings#ordinal-sorting-and-searching-in-collections): Collection classes such as System.Collections.Hashtable, System.Collections.Generic.Dictionary<TKey,TValue>, and System.Collections.Generic.List<T> have constructors that take a System.StringComparer parameter when the type of the elements or keys is string. In general, you should use these constructors whenever possible, and specify either StringComparer.Ordinal or StringComparer.OrdinalIgnoreCase.

Reply:
I don't get it, they're not for sorting, and it would only use `GetHasCode` and `Equals` inside, `GetHashCode` would be calculated by their chars, `Equals` would compare by their chars too.  Besides, we all know they're just ASCII strings.

Reply:
Makes sense. Probably no need to pass comparator then.